vts/trustedservices/trustedservices_grpc.go

```diff
@@ -105,6 +108,7 @@ func (o *GRPC) Init(v *viper.Viper, pm plugin.IManager[handler.IEvidenceHandler]
 	}
 
 	o.PluginManager = pm
+	o.EPluginManager = em
```

Comment:
As with the local var name above, the `E` prefix is unclear. `EndorsementPluginManager` or `EndPluginManager` or similar. And the evidence `PluginManager` should be renamed to match.